### DISCUSSION TITLE
Getting SSO Access Token

### DISCUSSION THREAD

**Discussion by GeoffreyParrier** [directus/directus#17960] [Archive]
Hi everyone,

I have a Directus with a Twitch SSO and I want an access to Twitch API

Is there anything I could do to get a twitch refresh or access token from Directus or do I need to get it from my front ? 

Thanks for your help

**Comment by aidenfoxx** [comment]
There is a PR open which would expose the access token in a hook on user authentication (`auth.update`). You can show your support for it here: https://github.com/directus/directus/pull/16812

**Reply by GeoffreyParrier** [reply]
Oh thanks a lot
